website/content/docs/concepts/domain-model/credential-libraries.mdx

```diff
@@ -39,25 +39,18 @@ A Vault credential library has the following additional attributes:
 
 ### Vault Credential Library Parameter Templating
 
-Some of the secrets engines that generate dynamic users for external systems provide the ability for Vault operators to customize how usernames are generated for those external systems.
-For more information, refer to [Username Templating](https://developer.hashicorp.com/vault/docs/concepts/username-templating/).
+Boundary operators may want to request credentials from a Vault path based on the user or account attributes that make the call.
```

Comment:
Both paths and request bodies (for `POST`) are supported, this makes it sound like only paths are supported.

Reply:
Thank you! I updated this to read:

> Boundary operators may want to request credentials from a Vault path or `POST` request body based on the user or account attributes that make the call.